docs/Development/ReleaseNotes.md

```diff
@@ -14,30 +14,6 @@ To check which release of VIC you are running:
 - For VIC 5 and later, type `vic_{classic,image}.exe -v`
 
 ------------------------------
-
```

Comment:
All the changes in this file should be rolled back. We also need to add a release note in the 5.1.0 in the `New Features` section.